sfputil/main.py

```diff
@@ -864,6 +880,100 @@ def on(port_name):
     """Enable low-power mode for SFP transceiver"""
     set_lpmode(port_name, True)
 
+# 'show loopback' subcommand
+@show.command()
+@click.option('-p', '--port', metavar='<port_name>', help="Display SFP low-power mode status for port <port_name> only")
```

Comment:
suggest adding a unit test to cover this new CLI

Reply:
Please check the UT result above